raiden-cli/src/controllers/channels.ts

```diff
@@ -1,6 +1,6 @@
 import { Request, Response, NextFunction } from 'express';
 import { ChannelState } from 'raiden-ts';
-import _ from 'lodash';
+import { map } from 'lodash';
```

Comment:
It should be:
```suggestion
import map from 'lodash/map';
```
As lodash isn't ES module by default (you'd need `lodash-es` for that), just destructuring won't get it tree-shaken, but importing the independent modules will.

Reply:
Oh, sorry. That was stupid. :see_no_evil:
Anyways this file does not exit anymore. Actually already when you reviewed. :thinking: